lnd.php

```diff
@@ -1,6 +1,6 @@
 <?php
-require 'vendor/autoload.php';
-
+//require '/lnd-php-rest/vendor/autoload.php';
+require dirname(__FILE__).'/../lnd-php-rest/vendor/autoload.php';
```

Comment:
are you sure this should be here when it is a composer package? 
can you show me another package that has such a require statement?